func_adl_xAOD/common/statement.py

```diff
@@ -153,6 +153,22 @@ def emit(self, e):
         e.add_line('{0} = {1};'.format(self._target.as_cpp(), self._value.as_cpp()))
 
 
+class decl_var_w_ctor:
```

Comment:
We need to understand if we really want a second way to declare variables. Could the ctor functionality be implemented elsewhere?